docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -123,17 +122,17 @@ Note that the detector does not detect black, perfect greys (greys where the red
 Viam [camera](/components/camera/) components can be [physical](/components/camera/webcam/) like the one already configured on the rover, or virtual.
 A virtual camera transforms the output from a physical camera.
 
-To view output from the color detector overlaid on images from a physical camera, you need to configure a [transform camera](/components/camera/transform/).
+To view output from the color detector overlaid on images from a physical camera, configure a [transform camera](/components/camera/transform/).
 
-Navigate to the **Components** tab in the Viam app and scroll to the **Create Component** section.
+Navigate to the **Components** tab in the Viam app and click the **Components** subtab.
 To create a [transform camera](/components/camera/transform/):
```

Comment:
```suggestion
```
This part's kinda redundant

Reply:
Oh I'm sorry my suggestion had "components" in it twice instead of "config" and then "components"! Sorry. Making new suggestion.